packages/vue-renderer/src/renderer.js

```diff
@@ -63,14 +63,30 @@ export default class VueRenderer {
     return context.renderScripts()
   }
 
+  getPreloadFiles(context) {
+    const legacyFiles = context.getPreloadFiles()
```

Comment:
Not sure if "legacy" is the best naming here :thinking: Maybe `transpiled` would fit better?

Reply:
Oh! Good point, didn't know that ☺️